pyrometheus/chem_expr.py

```diff
@@ -301,12 +300,13 @@ def rate_coefficient_expr(rate_coeff: ct.Arrhenius, t):
     a = rate_coeff.pre_exponential_factor
     b = rate_coeff.temperature_exponent
     t_a = rate_coeff.activation_energy/ct.gas_constant
-    if t_a == 0:
-        # Weakly temperature-dependent rate
-        return a * t**b
-    else:
-        # Modified Arrhenius
-        return p.Variable("exp")(np.log(a)+b*p.Variable("log")(t)-t_a/t)
+    return p.Variable("exp")(np.log(a)+b*p.Variable("log")(t)-t_a/t)
+#    if t_a == 0:
```

Comment:
Why is this commented out?

Reply:
I commented out the previous implementation, left it there as a reminder. I personally prefer having the same expression for both cases. But no strong opinion on this.

Reply:
I think that if we always return the full expression, we will always end up evaluating an `exp` with a call to a `log` inside, at least in Python, making the evaluation more expensive than it needs to be. Maybe these get removed when translating to OpenCL using Pytato, but we have users that work fully in Python.